Druid/DRMethod.class.st

```diff
@@ -9,6 +9,12 @@ Class {
 	#tag : 'CompilerCompiler'
 }
 
+{ #category : 'testing' }
+DRMethod class >> acceptsOpcode: anOpcode [
+
+	^false
+]
```

Comment:
This should not be the default in some superclass? 

Because now it's just defined in some of the DRInstruction